filesystem/lvm.ksy

```diff
@@ -0,0 +1,163 @@
+meta:
+  title: Logical Volume Manager
+  id: lvm
+  endian: le
+  encoding: ascii
+  application:
+    - linux
+    - grub2
+    - lvm tools
+    - libvslvm
+  license: GNU-FDL-1.3+
+doc: >
```

Comment:
This must be `|`, not `>`, or it will be parsed as one long line, not separate lines (as your formatting implies here).

Reply:
It's by design: I think that manual wrapping is evil, every modern text editor has automatical wrapping, which takes into account window width and text size.